pages/games/[gameId].tsx

```diff
@@ -147,156 +152,218 @@ function GamePage(): JSX.Element {
     const chainId = DEFAULT_CHAIN.id;
 
     return (
-      <VStack as="main" px={6} w="full">
-        <HStack align="start" py={16} px={20} w="100%">
-          <HStack flex="1">
-            <VStack align="start" justify="start" w="100%">
-              <Link
-                fontSize="sm"
-                fontWeight={300}
-                href={`${EXPLORER_URLS[chainId]}/address/${id}`}
-                isExternal
-                mb={2}
-                textDecoration={'underline'}
+      <Grid templateColumns="3fr 1fr" w="full" gridGap="5px">
+        <HStack spacing="5px">
+          <HStack
+            bg="cardBG"
+            p={8}
+            transition="background 0.3s ease"
+            w="100%"
+            spacing={12}
+          >
+            <AspectRatio ratio={1} w="100%" maxW="12rem">
+              <Image
+                alt="game emblem"
+                background="gray.400"
+                objectFit="cover"
+                src={image}
+                w="100%"
+                h="100%"
+              />
+            </AspectRatio>
+            <VStack spacing={4} align="flex-start">
+              <Heading
+                display="inline-block"
+                fontSize="40px"
+                fontWeight="normal"
+                lineHeight="40px"
+                _hover={{
+                  color: 'accent',
+                }}
               >
-                {shortenAddress(id)}
-              </Link>
-              <Heading fontSize="40px" fontWeight="normal" lineHeight="40px">
                 {name}
               </Heading>
-              <Text>{description}</Text>
+              <Text fontSize="xl" fontWeight={200} mb={2}>
+                {description}
+              </Text>
+              <HStack spacing={4}>
+                <Link
+                  fontSize="sm"
+                  href={`${EXPLORER_URLS[chainId]}/address/${id}`}
+                  isExternal
+                  fontWeight={300}
+                  mb={3}
+                  textDecoration={'underline'}
+                >
+                  {shortenAddress(id)}
+                </Link>
+                {isMaster && (
+                  <Button onClick={updateGameMetadata.onOpen} size="sm">
+                    <Flex align="center" gap={2}>
+                      <Image
```

Comment:
Looks like this image is removed in Ben's new design (looks like he also has the "e" lowercase)

Reply:
Removed in 5fff41d32f2580a90e7ca648147de5f3bd57535c